pkg/services/sqlstore/alert.go

```diff
@@ -15,14 +15,13 @@ import (
 var timeNow = time.Now
 
 func (ss *SQLStore) addAlertQueryAndCommandHandlers() {
-	bus.AddHandler("sql", SaveAlerts)
```

Comment:
Since removing this seems like it's not in use. Would be good to remove related code, such as the command https://github.com/grafana/grafana/blob/cb27c9cd6f1e3304af56b33ae46ab44762d23132/pkg/models/alert.go#L130-L136

And removing any other code referencing the command, if any.